bdk-ffi/src/lib.rs

```diff
@@ -39,6 +39,11 @@ pub struct ScriptAmount {
     pub amount: u64,
 }
 
+pub struct SentAndReceivedResult {
```

Comment:
I think it'd be a little clearer to call this `SendAndReceivedValues` instead of `Result` since result has special meaning in rust. Also I'd put this struct in `wallet.rs` since that's the only place it's used.

Reply:
Thanks for the feedback! 

I was unsure of what to name that struct so I did my best guess based any other types I could find that might be similar-ish in the existing codebase (like`TxBuilderResult`, etc), so it's good to get your thoughts on what to name it. 

Also thanks for the suggestion of putting the struct in `wallet.rs`.

I'll make these suggested changes and update the PR:
- `SentAndReceivedResult` -> `SentAndReceivedValues`
- move `SentAndReceivedValues` struct to `wallet.rs`

Reply:
No problem, I've also struggled to figure out what to call the types we use to replace the rust tuples!